docs/administrative-controls-for-smart-contracts.md

```diff
@@ -0,0 +1,25 @@
+---
+id: administrative-controls-for-smart-contracts
+title: Administrative controls for smart contracts
+sidebar_label: Administrative controls for smart contracts
```

Comment:
Let's make the `sidebar_label` a bit shorter, like `Contracts Admin Controls` or something in that nature. Make sure to have the update in both files : ) 

And let's keep the files naming shorter too (e.g. `contracts-controls.md`). It will be s easier to manage in the long-term.